### ISSUE TITLE
rewrite other shebangs

### ISSUE THREAD

**Issue by kapouer** [spotify/dh-virtualenv#305]
Hi,

using version https://github.com/spotify/dh-virtualenv/commit/ac6e1b19af57ac0aff6e63c67c9597a03d164a20
some scripts installed by setup.py have a [custom shebang in awx](https://github.com/ansible/awx/blob/66739a52d6ee051c688d5128948dea0f96f73794/tools/docker-compose/awx-manage#L1).
Once installed they are not rewritten by dh-virtualenv (because it searchs only for python and al.).

dh-virtualenv could either:
- search for other "absolutes" shebangs
- give a way to configure which interpreters must trigger a shebang rewrite (here it would be awx-python).

Thanks !

**Comment by jhermann** [comment]
This is what overrides in the rules file are for, allowing customization in the different build / install steps.

**Comment by kapouer** [comment]
Yes, but: rewriting a shebang is not that trivial, and is very related to virtualenv shortcomings (as virtualenv do rewrite *all* shebangs).

**Comment by jhermann** [comment]
> Yes, but: rewriting a shebang is not that trivial, and is very related to virtualenv shortcomings (as virtualenv do rewrite _all_ shebangs).

Fits my definition of trivial:

    python -m zipapp -p /bin/true -o out in
